src/improvements/template/EnableDeputyPauseModuleTemplate.sol

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {stdStorage, StdStorage} from "forge-std/Test.sol";
+import {VmSafe} from "forge-std/Vm.sol";
+
+import "forge-std/Test.sol";
+
+import {MultisigTask} from "src/improvements/tasks/MultisigTask.sol";
+import {ModuleManager} from "lib/safe-contracts/contracts/base/ModuleManager.sol";
+import {AddressRegistry} from "src/improvements/AddressRegistry.sol";
+
+/// @title EnableDeputyPauseModuleTemplate
+/// @notice Template contract for enabling a module in a Gnosis Safe
+contract EnableDeputyPauseModuleTemplate is MultisigTask {
+    using stdStorage for StdStorage;
+
+    /// @notice Module configuration loaded from TOML
+    address public newModule;
+
+    /// @notice Constant safe address string identifier
+    string constant _SAFE_ADDRESS = "FoundationOperationSafe";
+
+    /// @notice Gnosis Safe Sentinel Module address
+    address internal constant SENTINEL_MODULE = address(0x1);
+
+    /// @notice Gnosis Safe Module Mapping Storage Offset
+    uint256 public constant MODULE_MAPPING_STORAGE_OFFSET = 1;
+
+    /// @notice Gnosis Safe Module Nonce Storage Offset
+    bytes32 public constant NONCE_STORAGE_OFFSET = bytes32(uint256(5));
+
+    /// @notice Chain ID of the chain being modified
+    uint256 public immutable CHAIN_ID;
+
+    /// @notice Mainnet Chain ID
+    uint256 public immutable MAINNET_CHAIN_ID;
+
+    constructor() {
+        CHAIN_ID = block.chainid;
+        MAINNET_CHAIN_ID = getChain("mainnet").chainId;
+    }
+
+    /// @notice Returns the safe address string identifier
+    /// @return The string "DeputyPauseSafe"
+    function safeAddressString() public pure override returns (string memory) {
+        return _SAFE_ADDRESS;
+    }
+
+    /// @notice Returns the storage write permissions required for this task
+    /// @return Array of storage write permissions
+    function _taskStorageWrites() internal view override returns (string[] memory) {
+        // if mainnet is the chain being modified, this task will write to the
+        // LivenessGuard contract
+        // hardcode the chain id for mainnet to avoid making the function
+        // mutative and overriding the pure keyword
+
+        // this code does not work:
+        //   if (block.chainid == getChain("mainnet").chainId) {
+        // because the getChain function writes to storage, making this
+        // function mutative
+        // Instead, we write the chain id for mainnet in the constructor
+
+        string[] memory storageWrites;
+
+        if (CHAIN_ID == MAINNET_CHAIN_ID) {
+            storageWrites = new string[](2);
+            storageWrites[1] = "LivenessGuard";
+        } else {
+            // otherwise, we only need to write to the Safe
+            storageWrites = new string[](1);
+        }
```

Comment:
Sepolia also has the Liveness Guard
```suggestion
        if (CHAIN_ID == getChain("mainnet").chainId || getChain("sepolia").chainId) {
            storageWrites = new string[](2);
            storageWrites[1] = "LivenessGuard";
        } else {
            // otherwise, we only need to write to the Safe
            storageWrites = new string[](1);
        }
```

Reply:
Ah right the DPM goes on a foundation safe, which does not have the liveness guard on both mainnet and sepolia. So I think we can delete this altogether?